<launch>
  <node pkg="nodelet" type="nodelet" name="image_nodelet_manager"  args="manager" output="screen">
      <!-- load into nodelet for now, handle in the class by passing nodelet plugin name later -->
      <rosparam command="load" file="$(find drive_ros_image_recognition)/homography_matrix.yaml"/>
      <rosparam command="load" file="$(find drive_ros_image_recognition)/camera_matrix.yaml"/>
  </node>

  <node pkg="nodelet" type="nodelet" name="WarpImageNodelet" args="load drive_ros_image_recognition/WarpImageNodelet image_nodelet_manager" output="screen">
    <remap from="~/img_in" to="/camera1/image_raw"/>
  </node>
</launch>
